### ISSUE TITLE
Output column 2 and 3, values

### ISSUE THREAD

**Issue by fritzsedlazeck** [vibansal/HapCUT2#32]
Hi,
I did not see that documented, but some of my SNPs reported in the final file show a '-' in both 2nd and 3rd column and I was wondering what that means. 

examples:

> 352     1       0       chr1    199492  G       A       0/1:5:13:18:.:63,0,322  0       .       37.89
> 353     -       -       chr1    199493  A       G       0/1:4:14:18:.:34,0,350  1       .       3.01
> 354     1       0       chr1    199589  G       A       0/1:2:2:4:.:18,0,48     0       .       19.00
> 355     -       -       chr1    200930  G       A       0/1:4:20:24:.:16,0,504  1       .       3.01
> 356     1       0       chr1    201549  G       A       0/1:3:11:14:.:17,0,276  0       .       37.89

The parser to vcf translates them into 0|1 but I am not sure if that is correct...

Thanks
Fritz

**Comment by fritzsedlazeck** [comment]
Oh is it because of bad quality (last column)?

**Comment by vibansal** [comment]
Yes. These snps are unphased.


-------- Original message --------

**Comment by pjedge** [comment]
Yes, it is because of the default minimum mismatch quality value.

**Comment by fritzsedlazeck** [comment]
Thanks. I will report the parsing error in the other git repo.

**Comment by fritzsedlazeck** [comment]
Hey thanks for the quick replies. I wrote the the other parsing github, but they seem to be fine to report the snp as I indicated above. 
Anyways I wrote now my own parser (since that is not much work) that ignores these - - scenarios and also checks and updates only SNPs that are assigned to either haplotypes. So far it runs in 40sec on updating over 4 million SNPs using 2.5 million phased SNPs from your output. 

That code is accessible here: https://github.com/fritzsedlazeck/SURVIVOR

Thanks again for the fast response (again). 
Fritz
